quickstarts/aws/aws-smd/config.yml

```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist/) to learn 
+  more about New Relic monitoring and integration with AWS System Manager Distributor.
```

Comment:
I think we could get a little more information in here before tossing the user out to the docs site. We want users to understand the value provided by this quickstart before they need to open links to other pages.